src/components/tictactoe board/Board.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import Board from './Board'
+import { useTictactoe } from 'context/TictactoeContext'
+import { BrowserRouter } from 'react-router-dom'
+
+jest.mock('context/TictactoeContext', () => ({
+  ...jest.requireActual('context/TictactoeContext'),
+  useTictactoe: jest.fn(),
+}))
+
+test('renders Board component with initial values', () => {
```

Comment:
Can we write a test for functionality too?
Functionality written in Board component

Reply:
Done